Library/Formula/mpd.rb

```diff
@@ -30,6 +30,7 @@ class Mpd < Formula
   option "with-libvorbis", "Build with vorbis support (for Ogg encoding)"
   option "with-yajl", "Build with yajl support (for playing from soundcloud)"
   option "with-opus", "Build with opus support (for Opus encoding and decoding)"
+  option "with-mad", "Build with libmad support"
```

Comment:
You can leave out this line if you don’t provide an explanation beside “build with libmad support”; it’ll be automatically added with the `depends_on "libmad" => :optional`.